submissions/xT0PERx/Popup-style/index.html

```diff
@@ -1,137 +1,180 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="utf-8" />
-        <meta name="viewport" content="width=device-width,initial-scale=1" />
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <title>Popup-style xT0PERx</title>
-        <link rel="stylesheet" type="text/css" href="style.css" />
-    </head>
-    <body>
-        <nav class="container">
-            <ol>
-                <li>
-                    <a class="mail" href="#" tabindex="1"><p>Mail</p></a>
-                </li>
-                <li>
-                    <a class="images" href="#" tabindex="2"><p>Images</p></a>
-                </li>
-                <li class="main_tullbar">
-                    <input
-                        tabindex="3"
-                        id="first"
-                        class="check_input"
-                        type="checkbox"
-                    />
-                    <label class="menu" for="first"> </label>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Popup-style xT0PERx</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
+  <body>
+    <nav class="container">
+      <ol>
+        <li>
+          <a class="menu_items" href="#">Mail</a>
+        </li>
+        <li>
+          <a class="menu_items" href="#">Images</a>
+        </li>
+        <li class="main_tullbar">
+          <input id="first" class="check_input" type="checkbox" />
+          <label class="menu" for="first"> </label>
 
-                    <div class="icons">
-                        <div tabindex="4">
-                            <img src="images\apps.png" />
-                            <p>Apps</p>
-                        </div>
-                        <div tabindex="4">
-                            <img src="images\blogs.png" alt="" />
-                            <p>Blogs</p>
-                        </div>
-                        <div tabindex="4">
-                            <img src="images\calendar.png" />
-                            <p>Calendar</p>
-                        </div>
-                        <div tabindex="4">
-                            <img src="images\chat.png" alt="" />
-                            <p>Chat</p>
-                        </div>
+          <div class="icons">
```

Comment:
Why you don't use a list tag for icons inside popup?

Reply:
I think this is due to the fact that I taught the elements of the table at the beginning of the course, and at the end of the div and flexbox. Well, naturally, due to lack of experience, in the future I will do this kind of thing use a list tag.